src/components/views/elements/AppTile.tsx

```diff
@@ -548,12 +553,12 @@ export default class AppTile extends React.Component<IProps, IState> {
             maxMinButton = <AccessibleButton
                 className={
                     "mx_AppTileMenuBar_iconButton"
-                                    + (widgetIsMaximised
-                                        ? " mx_AppTileMenuBar_iconButton_minWidget"
-                                        : " mx_AppTileMenuBar_iconButton_maxWidget")
+                    + (widgetIsMaximised
+                        ? " mx_AppTileMenuBar_iconButton_minWidget"
+                        : " mx_AppTileMenuBar_iconButton_maxWidget")
```

Comment:
I'm a bit surprised the linter was okay with either of these, but whatever appeases it I suppose.